test/write-permissions.test.js

```diff
@@ -70,15 +70,21 @@ Object.keys(testAPIs).forEach(API => {
       rmrf.sync(dbPath)
       ipfsd = await startIpfs(API, config.daemon1)
       ipfs = ipfsd.api
-      orbitdb1 = new OrbitDB(ipfs, dbPath + '/1')
-      orbitdb2 = new OrbitDB(ipfs, dbPath + '/2')
+      orbitdb1 = await OrbitDB.createInstance(ipfs, {
+        directory: dbPath + '/1',
+        id: 'A'
```

Comment:
Do we need to specify the `id` here? If not, we could remove these to simplify the code further.

Reply:
Keystore makes an identity using the ipfs peer id, since they are the same the identities would be the same here, the optional id is to use a different id in identity creation. I would rather find a better way but this was quick

Reply:
Got it 👍 Let's go with this then.